java/README.md

```diff
@@ -31,20 +31,28 @@ PLAID_PRODUCTS=[PRODUCTS] \
 PLAID_COUNTRY_CODES='US' \
 java -jar target/quickstart-1.0-SNAPSHOT.jar server config.yml
 
-# Go to http://localhost:8080
+# Go to http://localhost:8000
 ```
 
+### Link token creation and server-side configuration.
+The recommended way to initialize Plaid Link is to pass Plaid Link initialization
+parameters server-side to `link/token/create`. The server then returns the link token,
+which the client can then use to initialize Plaid Link.
+
+When the client initializes Plaid Link with the link token, the Plaid Link
+initialization parameters associated with the Plaid Token will be applied.
+
 ## The OAuth redirect flow
-Some European institutions require an OAuth redirect authentication flow, where the end user is redirected to the bank’s website or mobile app to authenticate. For this flow, you should provide two additional configuration parameters, `PLAID_OAUTH_NONCE` and `PLAID_OAUTH_REDIRECT_URI`.
+Some European institutions require an OAuth redirect authentication flow,
+where the end user is redirected to the bank’s website or mobile app to
+authenticate. For this flow, you should provide an additional parameter,
+`PLAID_REDIRECT_URI`
```

Comment:
perhaps we should mention that they should provide EITHER the `PLAID_REDIRECT_URI` or the `ANDROID_PACKAGE_NAME`

Reply:
Wait nvm. I guess they'll always be using link-web here 🙈